ansible/roles/operator-pipeline/tasks/operator-hosted-pipeline-trigger.yml

```diff
@@ -0,0 +1,196 @@
+---
+- name: Create trigger enabling running Hosted pipeline via GitHub webhook
+  block:
+    - name: Create Hosted pipeline event listener
+      k8s:
+        state: present
+        namespace: "{{ oc_namespace }}"
+        definition:
+          apiVersion: triggers.tekton.dev/v1alpha1
+          kind: EventListener
+          metadata:
+            name: operator-hosted-pipeline-github-listener
+          spec:
+            serviceAccountName: pipeline
+            triggers:
+              - name: github-listener-prod
+                interceptors:
+                  - github:
+                      secretRef:
+                        secretName: github-webhook-secret
+                        secretKey: webhook-secret
+                      eventTypes:
+                        - pull_request
+                  - cel:
+                      filter: >-
+                        (
+                          body.repository.html_url.contains("prod")
```

Comment:
It's not likely the prod repo will contain `prod` in the name.

Could we instead create separate routes for prod and preprod so that we don't need to filter by the repo url? I'd feel less nervous about them accidentally triggering from the wrong source that way.